external_contact.go

```diff
@@ -26,6 +26,19 @@ func (c *WorkwxApp) GetExternalContact(externalUserid string) (*ExternalContactI
 	return &resp.ExternalContactInfo, nil
 }
 
+// BatchExternalContact 批量获取客户详情
+func (c *WorkwxApp) BatchExternalContact(userID, cursor string, limit int) ([]ExternalContactBatchInfo, string, error) {
```

Comment:
1. `Batch` is not a verb, use something like `BatchListExternalContacts` would be better IMO
2. `a, b Type` is not allowed in this repo (like described in [this article](http://www.yinwang.org/blog-cn/2014/04/18/golang)), duplicating the `string` part is fine for readability
3. returning three values is not ergonomic, make a wrapper struct for that is fine

Reply:
Like the following

```go
type BatchListExternalContactsResp struct {
    Result []ExternalContactBatchInfo
    NextCursor string
}
```

P.S. the `ExternalContactBatchInfo` name is very unfortunate but I cannot come up with a better one, so leave it alone for now...

Reply:
give a friendly remind, [Batch is a verb word](https://www.oxfordlearnersdictionaries.com/us/definition/english/batch_2)

Reply:
Hah, I forgot it!

Though the "batch" verb is more suited to "batching requests" and such, it doesn't have the "list" meaning implied.